<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inscription</class>
 <widget class="QWidget" name="Inscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_Pseudo_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Pseudonyme :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inscrire">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>S'inscrire</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_pseudo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sharecount</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_email">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_mdp">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse e-mail :</string>
   </property>
  </widget>
  <widget class="QLabel" name="in_erreur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
